src/oas.md

```diff
@@ -924,6 +924,7 @@ The path itself is still exposed to the documentation viewer but they will not k
 | <a name="path-item-ref"></a>$ref | `string` | Allows for a referenced definition of this path item. The value MUST be in the form of a URI, and the referenced structure MUST be in the form of a [Path Item Object](#path-item-object). In case a Path Item Object field appears both in the defined object and the referenced object, the behavior is undefined. See the rules for resolving [Relative References](#relative-references-in-api-description-uris). <br><br>_**Note:** The behavior of `$ref` with adjacent properties is likely to change in future versions of this specification to bring it into closer alignment with the behavior of the [Reference Object](#reference-object)._ |
 | <a name="path-item-summary"></a>summary | `string` | An optional string summary, intended to apply to all operations in this path. |
 | <a name="path-item-description"></a>description | `string` | An optional string description, intended to apply to all operations in this path. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation. |
+| <a name="path-item-additional-operations"></a>additionalOperations | Map[`string`, [Operation Object](#operation-object)] | A map of additional operations on this path. This map MUST NOT contain any entry matching (case-insensitive) any operation that can be defined on the parent path item object. |
```

Comment:
I'd place this after the last explicit method name `trace` to make clear what it is "additional" to.

Reply:
done. It kind of always bothers me that properties are not in alphabetical order on this spec, but now I know there's some kind of "logical order" going on.